cmd/common/oci.go

```diff
@@ -89,6 +93,12 @@ func NewRunCommand(rootCmd *cobra.Command, runtime runtime.Runtime, hiddenColumn
 
 			ops := make([]operators.DataOperator, 0)
 			for _, op := range operators.GetDataOperators() {
```

Comment:
Don't we have global parameters for layer operators? I suppose we'll need to add them later on.

Reply:
Not yet, but yeah, we should add them sooner or later.